lib/Doctrine/ORM/Exception/NotSupported.php

```diff
@@ -15,25 +15,23 @@ public static function create(): self
 
     public static function createForDbal3(string $context): self
     {
-        return new self(sprintf(
-            <<<'EXCEPTION'
+        $message = <<<'EXCEPTION'
 Context: %s
 Problem: Feature was deprecated in doctrine/dbal 2.x and is not supported by installed doctrine/dbal:3.x
 Solution: See the doctrine/deprecations logs for new alternative approaches.
-EXCEPTION,
-            $context
-        ));
+EXCEPTION;
+
+        return new self(sprintf($message, $context));
```

Comment:
Apparently, the way we used HEREDOC syntax in this file was incompatible with PHP 7.2.

Reply:
Yes, but you don't need to create a variable for this, having the comma on its own line should be enough, I'm going to amend this PR to fix that